packages/hasher.vm/tools/chocolateyuninstall.ps1

```diff
@@ -0,0 +1,7 @@
+$ErrorActionPreference = 'Continue'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'Hasher'
+$category = 'Forensic'
```

Comment:
We have similar tools in `Utilities` instead of `Forensic`.

Reply:
I change the category to `Utilities`